internal/pkg/addon/package.go

```diff
@@ -213,6 +215,60 @@ func (t *cfnTemplate) pkg(a *Addons) error {
 	return nil
 }
 
+// packageTransforms searches each node in nodes for the CFN
+// intrinsic function "Fn::Transform" with the "AWS::Include" macro. If it
+// dectects one, and the "Location" parameter is set to a local path, it'll
+// upload those files to S3. If node is a yaml map or sequence, it will
+// recursivly traverse those nodes.
+func (a *Addons) packageTransforms(nodes ...yaml.Node) error {
+	forNode := func(node yaml.Node) error {
```

Comment:
nit: Should we take as input a pointer to make it clear that we're mutating the internals
```suggestion
	pkg := func(node *yaml.Node) error {
```